addon/components/bs-accordion/item/title.hbs

```diff
@@ -19,9 +19,11 @@
 {{else}}
```

Comment:
Given markup between BS4 and BS5 is now much more aligned, doesn't it make sense to remove that if/else condition, and only apply conditionals where they apply (classes for the outer div and h2 AFAICT)?

Reply:
I thought about that as well. But would like to avoid the unnecessary `<div>` for BS5. And not rendering the wrapping element conditionally is very complex in current Ember.